common/libzkp/integration_test/testdata/phase2/witnesses/8.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
if we really want to add these 8 json files, we 'd better rename the dir from "phase2" to "euclidv2".

Reply:
As above